client/src/components/atoms/InlineComment.vue

```diff
@@ -0,0 +1,69 @@
+<template>
```

Comment:
The diff between this file and InlineCommentReply, make me think they could easily be the same component.  Given that multi-root components are a thing now, I'd also consider moving the rendering of comment replies part of this component (and recursively render InlineComment with mayne a "isReply" boolean prop)

Reply:
This has been refactored as suggested.